components/Comments.tsx

```diff
@@ -0,0 +1,41 @@
+import { Divider, Spacer, Text } from '@geist-ui/react';
+import { formatRelative, parseISO } from 'date-fns';
+
+import { Comment } from '../redux/types';
+import Person from './Person';
+
+interface CommentsParams {
+  comments?: Comment[];
```

Comment:
Since you assigned a default value, I think you don't need to double-check.

```suggestion
  comments: Comment[];
```